package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "NODE_ENV=test mocha src/server/test",
     "start": "node bin/server.js",
-    "prestart": "sequelize db:migrate"
+    "prestart": "sequelize db:migrate",
```

Comment:
You could also now refactor the `prestart` script to just call `npm run migrate` directly now so that you don't need to keep them in sync.